lib/vagrant-service-manager/services/openshift.rb

```diff
@@ -0,0 +1,24 @@
+module Vagrant
+  module ServiceManager
```

Comment:
Irrespective of whether docker is being listed or not, or a service (openshift) depends on docker, since we are enabling docker by default, its implicit. Why do we need to link it in dependency chain ?